test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -88,6 +88,7 @@
     ("aten::grid_sampler_2d_backward", datetime.date(2021, 10, 21)),
     ("aten::index_add.alpha", datetime.date(2021, 12, 31)),
     ("aten::index_add_.alpha", datetime.date(2021, 12, 31)),
+    ("aten::index_copy_", datetime.date(2022, 6, 31)),
```

Comment:
cc: @bdhirsh - Do you think the date is fine here?

Reply:
Ahhh!! Really sorry Brian, kinda lost these days :( I've fixed this now (the new date is 31st May), apologies again.

Reply:
Hey no worries at all :). I've been reviewing this PR pretty asynchronously, so that totally makes it easier to lose track of the current state of things (for me too).

Reply:
Thank you so much, @bdhirsh - for understanding. I appreciate it. Just a quick question, from the current failure:
```cpp
Broken ops: [
2022-02-04T20:03:18.9987343Z 	aten::_index_copy_(Tensor(a!) self, int dim, Tensor index, Tensor source) -> (Tensor(a!))
2022-02-04T20:03:18.9987404Z ]
```

Should I add `_index_copy_` to the list as well?